doc/upgrading.txt

```diff
@@ -1,8 +1,18 @@
 Upgrading
 =========
+Migrating from 9.x to 10.0
+-------------------------
+Python 3.5 or newer is required.
+
+The Python Beautiful Soup package is now required. A C compiler is not needed
+for building.
+
+The loginuserfield and loginpasswordfield entries in the authentication section
+of linkcheckerrc are now case-sensitive.
```

Comment:
AFAIU they were always case-sensitive.  They were used in a couple of places in the code, and while one of them handled them case-insensitively, the other one was always case-sensitive, making the overall behavior also case-sensitive.

What might have changed is the exact error you get if you get the case wrong.

Reply:
OK, so they used to match case-insensitively:
https://github.com/linkchecker/linkchecker/blob/71be9b941b2f2e101d5cbe9113913cca32a4fe27/linkcheck/htmlutil/formsearch.py#L107

But then posted case-sensitively:
https://github.com/linkchecker/linkchecker/blob/71be9b941b2f2e101d5cbe9113913cca32a4fe27/linkcheck/director/aggregator.py#L91-L92

Update the text to:

The loginuserfield and loginpasswordfield entries in the authentication section
of linkcheckerrc now match a login form case-sensitively.